docs/cli_README.md

```diff
@@ -5,28 +5,64 @@
 ### Installation:
 - Select, Download and Install latest binary from [Releases](https://github.com/fabric8-analytics/cli-tools/releases)
 
+#### curl
+
+- ##### For Linux
+```
+$. curl -s -L https://github.com/fabric8-analytics/cli-tools/releases/download/v0.0.1/crda_0.0.1_Linux_64bit.tar.gz | tar xvz -C .
```

Comment:
Below format to download latest works for me.

https://docs.github.com/en/github/administering-a-repository/linking-to-releases
```
/owner/name/releases/latest/download/asset-name.zip
```

```suggestion
$ curl -s -L https://github.com/fabric8-analytics/cli-tools/releases/latest/download/crda_0.0.1_Linux_64bit.tar.gz | tar xvz -C .
```

Reply:
Ack (dot removed)
- sticking to URL with fixed version as of now